.gitignore

```diff
@@ -21,7 +21,7 @@ node_modules/
 # Build folder
 dist/
 
-# Experimental Schemas
+# Utility testing output
 utils/output/**/*.*
```

Comment:
fix dis

Reply:
I realized I cant ignore the entire folder, i want the entire folder to exist, thats why theres a `.gitkeep`, I have just forgotten